drivers/can/can_handlers.c

```diff
@@ -48,7 +48,8 @@ static inline int z_vrfy_can_get_core_clock(const struct device *dev,
 static inline int z_vrfy_can_get_max_bitrate(const struct device *dev,
 					     uint32_t *max_bitrate)
 {
-	Z_OOPS(Z_SYSCALL_DRIVER_CAN(dev, get_max_bitrate));
+	/* Optional API function */
```

Comment:
Thanks for also double checking those verifier functions for the syscalls corresponding to the functions covered by the testcases